packages/plugins/inngest/src/schema-helpers.ts

```diff
@@ -20,6 +20,38 @@ import type {
   UseInngestEntityRecord,
 } from './types';
 
+/**
+ * getOperation
+ *
+ * Gets the operation document or execution params
+ *
+ * @param params
+ * @returns OperationTypeNode | 'unknown'
+ */
+export const getOperation = (params: OnExecuteEventPayload<ContextType>) => {
```

Comment:
Instead of having `getOperation` and `getOperationName`, we can call `getOperationAST` once and get both type and name, no?

Reply:
I think so, yes. I am refactoring and also adding tests for `schema-helpers` as I realized I left these out.

Reply:
I've changes to use just `getOperationAST` but when I tried to unify the name and operation, some of my even handles failed as the event name changed. I'll look more closely and see if I can eliminate duplicated `getOperationAST` calls.

Reply:
I unified the getOperation and getOperationName to return operationName and operationType to reduce `getOperationAST` calls.